### PR TITLE
Add r-suppdists

### PR DESCRIPTION
**Pull request by qwhelan**
Add the ```SuppDists``` R package from CRAN.

### CODE REVIEW

File: recipes/r-suppdists/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix  # [win64]
+    - m2w64-toolchain  # [win64]
+    - gcc  # [not win]
+  run:
+    - r-base
```
Comment (bgruening) [inline recipes/r-suppdists/recipe/meta.yaml]: please add libgcc not not-win.

**Comment by qwhelan** [comment]
@bgruening Apologies for the delay - was on vacation.

Incorporated your ```libgcc``` change, added you and @johanneskoester as maintainers, and rebased.

**Comment by bgruening** [comment]
LGTM. ping @isuruf :)

File: recipes/r-suppdists/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix  # [win64]
+    - m2w64-toolchain  # [win64]
+    - libgcc  # [not win64]
```
Comment (isuruf) [inline recipes/r-suppdists/recipe/meta.yaml]: This should be moved to `run` and 
```
- gcc  # [not win]
```
added to build

**Comment by qwhelan** [comment]
@isuruf  Done.

**Pull request merged**